code/modules/cargo/packs.dm

```diff
@@ -1016,6 +1024,55 @@
 /datum/supply_pack/materials
 	group = "Canisters & Materials"
 
+/datum/supply_pack/materials/diamond20
+	name = "20 Diamond Sheets"
+	desc = "Create with 20 sheets of diamonds. Usefull for M.R.S.P.A.C.M.A.N.-type portable generators."
+	cost = 15000 // 15000 worth of diamonds
+	contains = list(/obj/item/stack/sheet/mineral/diamond/twenty)
+	crate_name = "diamond sheets crate"
+
+/datum/supply_pack/materials/gold20
+	name = "20 Gold Ingot"
+	desc = "Create with 20 ingot of gold. For when people want that golden cargonia."
+	cost = 5000 // 3750 worth of gold
```

Comment:
This is either too high an amount of gold or too cheap of a crate. You can decide which. At a cost of 5000, it can be bought at the round's start and either bankrupt cargo or springboard science, either of which gives miners less of a reason to do their actual job.